src/App.tsx

```diff
@@ -1,15 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import Title from './components/Title';
 import AddItemForm from './components/AddItemForm';
-import TaskDashboard from './components/TaskDashboard';
+import TaskDashboard, { Task } from './components/TaskDashboard';
 
 function App() {
+  const [useListTask, setListTask] = useState<Task[]>([]);
```

Comment:
esse nome nao esta de acordo com boas praticas de react. o prefixo `use` é utilizado para custom hooks [veja](https://pt-br.reactjs.org/docs/hooks-custom.html#extracting-a-custom-hook).
```suggestion
  const [taskList, setTaskList] = useState<Task[]>([]);
```